### PR TITLE
Add option to exclude selected resources on export

### PR DESCRIPTION
**Pull request by KoBeWi**
Another part of https://github.com/godotengine/godot-proposals/issues/1454
image

### CODE REVIEW

**Comment by MCrafterzz** [comment]
I don't know if its already a feature but how about an option to not export unused resources (3d models, textures etc)?

**Comment by KoBeWi** [comment]
> I don't know if its already a feature but how about an option to not export unused resources (3d models, textures etc)?

Open a separate proposal for that. Also it's not easy to determine which resources are "used".

File: editor/project_export.cpp [hunk 1]
```diff
@@ -1029,6 +1029,7 @@ ProjectExportDialog::ProjectExportDialog() {
 	export_filter->add_item(TTR("Export all resources in the project"));
 	export_filter->add_item(TTR("Export selected scenes (and dependencies)"));
 	export_filter->add_item(TTR("Export selected resources (and dependencies)"));
+	export_filter->add_item(TTR("Exclude selected resources"));
```
Comment (Calinou) [inline editor/project_export.cpp:1032]: This description is quite long, but more explicit:

```suggestion
	export_filter->add_item(TTR("Export all resources in the project except resources checked below"));
```

**Review by Calinou — CHANGES_REQUESTED** [review]

**Review by Calinou — APPROVED** [review]

File: editor/project_export.cpp [hunk 2]
```diff
@@ -1029,6 +1029,7 @@ ProjectExportDialog::ProjectExportDialog() {
 	export_filter->add_item(TTR("Export all resources in the project"));
 	export_filter->add_item(TTR("Export selected scenes (and dependencies)"));
 	export_filter->add_item(TTR("Export selected resources (and dependencies)"));
+	export_filter->add_item(TTR("Export all resources in the project except resources checked below")));
```
Comment (akien-mga) [inline editor/project_export.cpp:1032]: Too many parentheses :)

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**